tests/embedding/test_app.py

```diff
@@ -28,23 +27,17 @@ def test_app_save_open(embedded_app):
 
     embedded_app.DataModel.Project.Name = "PROJECT 1"
     tmpname = tempfile.mktemp()
-    project_file = f"{tmpname}.mechdat"
-    project_files = f"{tmpname}_Mech_Files"
+    project_file = os.path.join(tmp_path, f"{tmpname}.mechdat")
     embedded_app.save_as(project_file)
-    try:
-        embedded_app.new()
-        embedded_app.open(project_file)
-        assert embedded_app.DataModel.Project.Name == "PROJECT 1"
-        embedded_app.DataModel.Project.Name = "PROJECT 2"
-        embedded_app.save()
-        embedded_app.new()
-        embedded_app.open(project_file)
-        assert embedded_app.DataModel.Project.Name == "PROJECT 2"
-        embedded_app.new()
-    except:
-        # clean up the project files
-        shutil.rmtree(project_files)
-        os.remove(project_file)
+    embedded_app.new()
```

Comment:
Should I still keep the try except, but have "pass" in except instead of the folder & file removal?

Reply:
@klmcadams the `except` block was only necessary to not leak files during test runs, its not needed anymore